ci_checks.sh

```diff
@@ -32,10 +32,8 @@ cargo install --locked --debug forc-explore &&
 cargo install --locked --debug --path ./scripts/mdbook-forc-documenter &&
 forc build --path sway-lib-core &&
 forc test --path sway-lib-core &&
-forc build --path sway-lib-std &&
-forc test --path sway-lib-std &&
```

Comment:
Because they were duplicated=)
<img width="327" alt="image" src="https://user-images.githubusercontent.com/18346821/236175728-66495141-06d1-4504-9270-c85268dac349.png">